### ISSUE TITLE
7zip Easy Unzip Improvements

### ISSUE THREAD

**Issue by nadrojisk** [mandiant/VM-Packages#1018]
### Details

I suggest:

- At the moment there is a handy unzip with "infected" right click context menu that utilizes 7zip. I suggest changing it to the following:
```
 VM-Add-To-Right-Click-Menu $toolName 'unzip "infected"' "`"$7zExecutablePath`" x -pinfected `"%1`"" "$executablePath" -extension $extension
```
- I suggest swapping `e` to `x` as it extract with full paths. Sometimes there are nested directories within a ZIP and it can be cumbersome if those are just flattened upon extracting. 
- I think it would also be great to extract the archive into a directory with its same name similar to how when you right click an archive it says extract to `[Archive Name]\`

**Comment by mr-tz** [comment]
Agreed on using `x`. No hard feelings on adding an outer directory.

**Comment by nadrojisk** [comment]
> Agreed on using `x`. No hard feelings on adding an outer directory.

No hard feelings at all :) Thanks for adding the x change though, you all are great!
